### DISCUSSION TITLE
Streaks info for numeric habits

### DISCUSSION THREAD

**Discussion by ghost** [iSoron/uhabits#1114] [Feature Requests]
All the yes-no habits have a section that gives you info on the best streaks. Would you please consider adding the same section for numeric habits as well? It should be simply based on the number of times targets were met in a row.

Thank you for the latest update, by the way, it's brilliant!

**Comment by VA-MMXXII** [comment]
Seems like a good idea!

**Comment by baincd** [comment]
I am really happy that measurable habits were added.  But I do feel like there is a gap in that measurable habits do not track streaks like yes-no habits.  

Just a guess, but I think this is related and may require #1180.

**Comment by SimonLammer** [comment]
I'd also really appreciate streaks for measurable habits. That way I could have small widgets for my streaks on my home screen to motivate me (as I already can for binary habits).

**Comment by teckwarz** [comment]
@iSoron Please check whether you approve this feature. This is one feature which I want in this app myself. If you could share some useful code pointers for streaks calculation and measurable habits, I could contribute for implementing this feature.

**Reply by teckwarz** [reply]
@iSoron Noticed that the data for streak is already available and removing this line https://github.com/iSoron/uhabits/blob/fc9cc423d06c9c83ee14dc9ae4bdeb0781a93a5c/uhabits-android/src/main/java/org/isoron/uhabits/activities/habits/show/ShowHabitView.kt#L57 shows the habit streak.

But difference I noticed is that it considers the entries for streak and doesn't consider whether the user reached the goal of the day for the streak calculation

**Comment by teckwarz** [comment]
Raised PR for above feature: https://github.com/iSoron/uhabits/pull/2059

**Reply by teckwarz** [reply]
Good news, the PR for the feature had been merged and the "Best Streaks" for numerical habits should be available as part of next update ✌️😇

**Reply by ZornHadNoChoice** [reply]
Thank you, I can't wait for this! Is there a streaks widget too?
